Frends.ODBC.ExecuteQuery/Frends.ODBC.ExecuteQuery/Definitions/Input.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.ODBC.ExecuteQuery.Definitions;
+
+/// <summary>
+/// Input parameters.
+/// </summary>
+public class Input
+{
+    /// <summary>
+    /// Connection string.
+    /// </summary>
+    /// <example>Driver={driver};Server=127.0.0.1,1433; Database=UnitTests;DSN=ODBC_testDB;Uid=sa;Pwd=yourStrong!Password;</example>
+    [PasswordPropertyText]
```

Comment:
There is a bug in Frends which sometimes makes this password protected field into a expression by default. This can be fixed in the Task by adding annotation [DisplayFormat(DataFormatString = "Text")]. Please add that.